src/Proposal.ts

```diff
@@ -86,9 +87,11 @@ export default class Proposal {
       : this.pod.threshold;
     this.safeTransaction = safeTransaction;
     this.rejectTransaction = rejectTransaction;
+    this.status = status;
 
-    if (podNonce === this.id) this.status = 'active';
-    if (podNonce > this.id) this.status = 'executed';
+    // TODO: differentiate between passed and executed proposals
+    if (podNonce === this.id) this.status = 'executed';
```

Comment:
I don't think this is correct, it should still be 'active' if the nonce === the proposal ID. I think you can keep line 94 as is though.